tools/plugins/install_cxx_tools.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -x
+
+. /vagrant/conf.sh
+
+set -e
+
+if [ ! -f /etc/debian_version ]; then
+    echo "Skipping $0"
+    exit 0
+fi
+
+apt -y install cmake g++ libzmq-dev libzmqpp-dev libcrypto++-dev liblog4cxx-dev
```

Comment:
I didn't notice this before, but the other SDKs are using libzmq3-dev. Is there a reason for using libzmq-dev? apt is telling me they conflict.